### PR TITLE
Adds SCP-513

### PR DESCRIPTION
**Pull request by steamp0rt**
- [x] Paranoia messages
- [x] Waking up the victims
- [x] Brain damage
- [x] Actual hallucinations

### CODE REVIEW

File: code/modules/SCP/SCPs/SCP-513.dm [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		to_chat(M, "<span class='danger'><i>\The [src] rings, sending chills to your very bone.</i></span>")
+		M << pick('sound/scp/spook/Bell2.ogg', 'sound/scp/spook/Bell3.ogg')
+		if(!(M in victims))
+			victims += M
+			brain_damage_timing[M] = world.time + rand(3000, 6500)
+
+/obj/item/scp513/pickup(mob/living/user)
+	. = ..()
+	if(user.a_intent == I_HURT)
+		to_chat(user, "<span class='danger'><b><i>You accidentaly ring \the [src]!</i></b></span>")
```
Comment (kachnov) [inline code/modules/SCP/SCPs/SCP-513.dm]: grammar

File: code/modules/SCP/SCPs/SCP-513.dm [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		ring(user)
+
+/obj/item/scp513/attack_self(mob/living/user)
+	ring(user)
+
+/obj/item/scp513/Process()
+	for(var/mob/living/carbon/M in victims)
+		if(prob(0.5))
```
Comment (kachnov) [inline code/modules/SCP/SCPs/SCP-513.dm]: seems kind of low, won't this take 400 seconds to see it once?

**Comment by kachnov** [comment]
oof, meant to request changes. Can you fix those?

File: code/modules/SCP/SCPs/SCP-513.dm (hunk 1 above)
Comment (steamp0rt) [reply code/modules/SCP/SCPs/SCP-513.dm]: What should I change it to?

File: code/modules/SCP/SCPs/SCP-513.dm (hunk 2 above)
Comment (steamp0rt) [reply code/modules/SCP/SCPs/SCP-513.dm]: will bump it up to 2.5?

File: code/modules/SCP/SCPs/SCP-513.dm [hunk 3]
```diff
@@ -28,15 +28,15 @@
...
+		to_chat(user, "<span class='danger'><b><i>You accidentally ring \the [src]!</i></b></span>")
 		ring(user)
 
 /obj/item/scp513/attack_self(mob/living/user)
 	ring(user)
 
 /obj/item/scp513/Process()
 	for(var/mob/living/carbon/M in victims)
-		if(prob(0.5))
+		if(prob(2.5))
```
Comment (JakeDaBoss) [inline code/modules/SCP/SCPs/SCP-513.dm]: Prob 2.5, that's a probability of 2.5% out of 100, isn't it?

**Comment by JakeDaBoss** [comment]
prob(2.5) is 2.5% out of 100%, isn't it?

File: code/modules/SCP/SCPs/SCP-513.dm (hunk 3 above)
Comment (steamp0rt) [reply code/modules/SCP/SCPs/SCP-513.dm]: Yep

**Comment by steamp0rt** [comment]
There's an SCP-513-1 that was dug up on the discord. I was initially planning on using alternate appearances, but the bay system for it is weird, plus it seems overkill. I could probably use client.images, if I can figure out how to use that

File: code/modules/SCP/SCPs/SCP-513.dm (hunk 3 above)
Comment (JakeDaBoss) [reply code/modules/SCP/SCPs/SCP-513.dm]: Explain the probability, and what it's meant to do if you could. Once I have a grasp on that function, then I can give you a pass or suggestion to fix.

**Comment by JakeDaBoss** [comment]
Check into how SCP-106 does it. We had to override the mob system, too.

**Comment by JakeDaBoss** [comment]
Make sure the mob can't be attacked as well.

**Comment by steamp0rt** [comment]
@JakeDaBoss I can't seem to find the 106 code?

**Comment by JakeDaBoss** [comment]
Scp-106 or scp106.dm

**Comment by LYNXEMS** [comment]
code/modules/SCP/SCPs/SCP-106.dm

**Comment by steamp0rt** [comment]
OH, it's on experimental branch. I was looking at master.

**Comment by steamp0rt** [comment]
Ok, @JakeDaBoss I rebased it to experimentral branch

**Comment by JakeDaBoss** [comment]
Yep. Saw. Cheers. How is 513-1 going?

**Comment by steamp0rt** [comment]
Looking at 106 code rn

File: code/modules/SCP/SCPs/SCP-513.dm [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+	var/static/list/spook_messages = list("You see a disturbing entity lingering in your peripheral vision", "You swear you can see an abomination lurking...",
+		"A strange entity stares at you, and sends chiils to your very core.")
+
+/obj/item/scp513/Initialize()
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/scp513/proc/ring(mob/living/user)
+	if(user in victims)
+		to_chat(user, "<span class='notice'>I rung it once, and I felt terrible. Why the hell would I that again?!</span>")
```
Comment (kachnov) [inline code/modules/SCP/SCPs/SCP-513.dm]: typo

Comment (kachnov) [inline code/modules/SCP/SCPs/SCP-513.dm]: typo

**Comment by steamp0rt** [comment]
It works now

**Comment by JakeDaBoss** [comment]
Address the typo fixes from Kachnov and then we should be ready to release it under alpha. Great work!

**Comment by JakeDaBoss** [comment]
Also requires changelog.

**Comment by steamp0rt** [comment]
@JakeDaBoss I did change rung to rang, review is still there for some reason. And I'll add a changelog tomorrow, trying to sleep rn

**Comment by steamp0rt** [comment]
NVM just realized I forgot to push my commit

**Comment by kachnov** [comment]
@steamp0rt I was referring to the missing verb in " Why the hell would I [do?] that again"

**Comment by steamp0rt** [comment]
Oh whoops

File: code/modules/SCP/SCPs/SCP-513.dm [hunk 5]
```diff
@@ -15,9 +15,6 @@
 	START_PROCESSING(SSobj, src)
 
 /obj/item/scp513/proc/ring(mob/living/user)
-	if(user in victims)
-		to_chat(user, "<span class='notice'>I rung it once, and I felt terrible. Why the hell would I that again?!</span>")
-		return
```
Comment (JakeDaBoss) [inline code/modules/SCP/SCPs/SCP-513.dm]: Why dod you remove these?

Comment (steamp0rt) [reply code/modules/SCP/SCPs/SCP-513.dm]: I moved it to attack_hand, so the message won't show if someone who has already rung it picks it up on harm intent

**Pull request merged**